core/modules/views/views.module

```diff
@@ -477,27 +477,42 @@ function views_menu_alter(&$callbacks) {
  * called in order to 'load' a views argument; primarily it
  * will be used to perform validation.
  *
- * @param $value
+ * @param string $value
  *   The actual value passed.
- * @param $name
+ * @param string $name
  *   The name of the view. This needs to be specified in the 'load function'
  *   of the menu entry.
- * @param $display_id
+ * @param string | array $display_id
  *   The display id that will be loaded for this menu item.
```

Comment:
This comment should be probably changed to use the plural, either as *The display IDs that …* or *The display ID(s) that …*.

Reply:
Thanks for the suggestion! "The display ID(s)" sounds good.